babel/src/main/java/org/apache/calcite/sql/babel/SqlDateTimeAtTimeZone.java

```diff
@@ -30,32 +29,46 @@
 
 import java.util.List;
 
+/**
+ * A {@code SqlDateTimeAtTimeZone} is an AST node that describes
+ * the date time expression of At Time Zone.
+ */
 public class SqlDateTimeAtTimeZone extends SqlCall implements SqlExecutableStatement {
   public static final SqlSpecialOperator OPERATOR =
       new SqlSpecialOperator("AT TIME ZONE", SqlKind.OTHER);
 
   private final SqlNode dateTimePrimary;
-  private final SqlIdentifier timeZoneValue;
+  private final SqlNode displacementValue;
 
+  /**
+   * Creates a {@code SqlTranslateUsingCharacterSet}.
+   *
+   * @param pos  Parser position, must not be null
+   * @param dateTimePrimary  SqlNode, contains the time to be transformed
+   * @param displacementValue SqlNode, contains the Displacement to the time
```

Comment:
nit: missing space

Reply:
Could you please be more specific on the "missing space" comment?

Reply:
pos and dateTimePrimary have two spaces after them. displaceValue only has one.

Reply:
I see, thanks for pointing out. Fixed in https://github.com/googleinterns/calcite/pull/91/commits/7cb7b83866002fdeda35007260c989d60c75e21a